tests/test_device_tracker.py

```
@@ -268,6 +273,177 @@ def test_handle_coordinator_update_entry_present(
+    make_config_entry: Callable[..., MockConfigEntry],
+    fake_reg_factory: Any,
+) -> None:
+    """Auto-discovered trackers should be enabled when HA can link a MAC device."""
+    ent = _make_scanner_entity(
+        coordinator=coordinator,
+        make_config_entry=make_config_entry,
+        coordinator_data={"arp_table": []},
+    )
+    ent.hass = ph_hass
+    device_reg = fake_reg_factory(device_exists=True, device_id="existing-device")
+    monkeypatch.setattr(dt_mod, "async_get_dev_reg", lambda hass: device_reg)
```

Ruff (0.15.18)

[warning] Unused lambda argument: `hass`

(ARG005)